redfish/rules/dell_redfish_1.17.0.yml

```diff
@@ -0,0 +1,431 @@
+Traverse:
+  Root: /redfish/v1
+  Excludes:
+  - /JsonSchemas
+  - /Accounts
+  - /Certificates
+  - /Jobs
+  - /Logs
+  - /Registries
+  - /Roles
+  - /Sessions
+  - /Settings
+  - /AccountService
+  - /CertificateService
+  - /EventService
+  - /JobService
+  - /LicenseService
+  - /LogServices
+  - /SessionService
+  - /TaskService
+  - /TelemetryService
+  - /UpdateService
+  - /Power/
+  - /Power#/
+  - /Thermal#/
+  - /Assembly#/
+  - /redfish/v1/Chassis/$
+Metrics:
+- Path: /redfish/v1/Chassis/System.Embedded.1/NetworkAdapters/NIC.Embedded.1
+  Properties:
+  - Name: chassis_systemembedded1_networkadapters_nicembedded1_status_health
+    Pointer: /Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_networkadapters_nicembedded1_status_state
+    Pointer: /Status/State
+    Type: state
+- Path: /redfish/v1/Chassis/System.Embedded.1/NetworkAdapters/NIC.Embedded.1/NetworkDeviceFunctions/{nic-dev-id}

```

Comment:
Do not include `-` in placeholders.

Reply:
I have reworked the rules for redfish v1.17 based on your comments.